src/bin/heroku-downloader.js

```diff
@@ -0,0 +1,28 @@
+#!/usr/bin/env node
+import spider from '../lib/spider.js';
+import download_raw_all_publisher_feeds from '../lib/download-raw.js';
+import Settings from '../lib/settings.js';
+import tls from 'tls';
+import Sentry from '@sentry/node';
+
+
+tls.DEFAULT_MIN_VERSION = Settings.tlsDefaultMinimumVersion;
+
+/* This heroku worker is intended as to be scheduled */
+
+if (process.env.SENTRY_DSN){
+  Sentry.init({ dsn: process.env.SENTRY_DSN });
+}
+
+(async() => {
+
+  console.log("-- Starting spider --");
+  await spider(Settings.spiderDataCatalogStartURL);
```

Comment:
If the intention here is that all data catalog work should be done before raw spiders start (which seems sensible) then won't https://github.com/openactive/conformance-services/blob/master/src/lib/spider.js#L31 and https://github.com/openactive/conformance-services/blob/master/src/lib/spider.js#L41 make that not the case?